beluga_example/README.md

```diff
@@ -42,6 +44,10 @@ This package contains example launch files that demonstrate the use of Beluga-ba
     rosrun teleop_twist_keyboard teleop_twist_keyboard
     ```
 
+    Note that this example uses [Flatland](https://flatland-simulator.readthedocs.io) for simulation.
+    A Flatland source installation is provisioned in development containers.
+    You will have to provision one yourself to run this elsewhere.
```

Comment:
nit^2:

```suggestion
    > [!IMPORTANT]
    > Note that this example uses [Flatland](https://flatland-simulator.readthedocs.io) for simulation.
    > A Flatland source installation is provisioned in development containers.
    > You will have to provision one yourself to run this elsewhere.
```

Reply:
Hmm, nevermind, it doesn't work inside a list :disappointed:

Reply:
That's a bummer. Reverted in https://github.com/Ekumen-OS/beluga/pull/361/commits/9cc2dd04c3516e53ad6741d30aa6561c9ef26997.

At least our quickstart does have the pretty admonitions.